src/schema.graphql

```diff
@@ -1,649 +1,655 @@
-  level: Long!
-  expiredBlockIndex: Long!
-  startedBlockIndex: Long!
-  receivedBlockIndex: Long!
-  rewardLevel: Long!
-  claimableBlockIndex: Long!
-}
-
-type MonsterCollectionStatusType {
-  fungibleAssetValue: FungibleAssetValueType!
-  rewardInfos: [MonsterCollectionRewardInfoType]
-  tipIndex: Long!
-  lockup: Boolean!
-}
-
-type NodeExceptionType {
-  code: Int!
-  message: String!
-}
-
-type NodeState {
-  preloaded: Boolean!
-}
-
-type NodeStatusType {
-  bootstrapEnded: Boolean!
-  preloadEnded: Boolean!
-  tip: BlockHeader!
-  topmostBlocks(limit: Int!, offset: Int = 0, miner: Address): [BlockHeader]!
-  stagedTxIds(address: Address): [TxId]
-  genesis: BlockHeader!
-  isMining: Boolean!
-  appProtocolVersion: AppProtocolVersionType
-}
-
-enum NotificationEnum {
-  REFILL
-  HAS
-  COMBINATION_EQUIPMENT
-  COMBINATION_CONSUMABLE
-  BUYER
-  SELLER
-}
-
-type NotificationType {
-  type: NotificationEnum!
-  message: String
-}
-
-type OrderDigestType {
-  orderId: Guid!
-  tradableId: Guid!
-  startedBlockIndex: Int!
-  expiredBlockIndex: Int!
-  sellerAgentAddress: Address!
-  price: String!
-  combatPoint: Int!
-  level: Int!
-  itemId: Int!
-  itemCount: Int!
-}
-
-type PeerChainStateQuery {
-  state: [String]!
-}
-
-type PreloadStateExtraType {
-  type: String!
-  currentCount: Long!
-  totalCount: Long!
-}
-
-type PreloadStateType {
-  currentPhase: Long!
-  totalPhase: Long!
-  extra: PreloadStateExtraType!
-}
-
-type PrivateKeyType {
-  hex: ByteString!
-  publicKey: PublicKeyType!
-}
-
-type ProtectedPrivateKeyType {
-  address: Address!
-}
-
-scalar PublicKey
-
-type PublicKeyType {
-  hex(compress: Boolean): ByteString!
-  address: Address!
-}
-
-type QuestListType {
-  completedQuestIds: [Int!]!
-}
-
-type RankingInfoType {
-  exp: Long!
-  level: Int!
-  armorId: Int!
-  updatedAt: Long!
-  stageClearedBlockIndex: Long!
-  agentAddress: Address!
-  avatarAddress: Address!
-  avatarName: String!
-}
-
-type RankingMapStateType {
-  address: Address!
-  capacity: Int!
-  rankingInfos: [RankingInfoType!]!
-}
-
-type RpcInformationQuery {
-  totalCount: Int!
-  clients: [Address]!
-}
-
-type ShardedShopStateV2Type {
-  address: Address!
-  orderDigestList(id: Int, maximumPrice: Int): [OrderDigestType]!
-}
-
-type ShopItemType {
-  sellerAgentAddress: Address!
-  sellerAvatarAddress: Address!
-  productId: Guid!
-  price: String!
-  itemUsable: ItemUsableType
-  costume: CostumeType
-}
-
-type ShopStateType {
-  address: Address!
-  products(id: Int, itemSubType: ItemSubType, maximumPrice: Int): [ShopItemType]!
-}
-
-type SkillType {
-  id: Int!
-  elementalType: ElementalType!
-  power: Int!
-  chance: Int!
-}
-
-type StakeAchievementsType {
-  achievementsByLevel(level: Int!): Int!
-}
-
-type StakeRegularRewardInfoType {
-  itemId: Int!
-  rate: Int!
-}
-
-type StakeRegularRewardRowType {
-  level: Int!
-  requiredGold: Long!
-  rewards: [StakeRegularRewardInfoType!]!
-}
-
-type StakeRegularRewardSheetType {
+schema {
```

Comment:
It isn't a suggestion to do in this pull request. But it seems we should configure the line ending rule with editorconfig or else. In my memory, the change of all lines appeared sometimes and it was because of the line ending (CRLF -> LF or LF -> CRLF).